recipes/intel-repack/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set version = "2019.5" %}
+{% set buildnum = "281" %}
+
+# use this if our build script changes and we need to increment beyond intel's version
+# {% set dstbuildnum='4' %}
+{% set openmp_version = version %}
+  # 117 was intel's base build number.  We're up 1 after making the license file not read-only
+{% set openmp_buildnum = buildnum %}
+
+package:
+  name: mkl
+  version: {{ version }}
+
+source:
+  - url: https://anaconda.org/intel/mkl/{{ version }}/download/{{ target_platform }}/mkl-{{ version }}-intel_{{ buildnum }}.tar.bz2
```

Comment:
@CJ-Wright will the bot understand how to update this version?

Reply:
Maybe, I'm concerned about the build number in the url

Reply:
Right ok. Let's make an issue on the bot scripts repo to make sure this can work.